sdk/core/azure-core/inc/azure/core.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Add CL entry similar to https://github.com/Azure/azure-sdk-for-c/blob/master/CHANGELOG.md#other-changes-and-improvements
> Add high-level and simplified az_core.h and az_iot.h files for simpler include experience for customers.